orbit/models/ktrlite.py

```diff
@@ -255,10 +259,15 @@ def get_gap_between_dates(start_date, end_date, freq):
         return gap
 
     @staticmethod
-    def _set_knots_tp(knots_distance, cutoff):
-        # start in the middle
-        knots_idx_start = round(knots_distance / 2)
-        knots_idx = np.arange(knots_idx_start, cutoff, knots_distance)
+    def _set_knots_tp(knots_distance, cutoff, knot_location):
+        if knot_location == 'mid_point':
+            # start in the middle
+            knots_idx_start = round(knots_distance / 2)
+            knots_idx = np.arange(knots_idx_start, cutoff, knots_distance)
+        elif knot_location == 'end_point':
```

Comment:
there should be a better way to calculate, e.g. num = 1/span? and also please put something more explicit like num_of_knot ?